blockchain/beacon/Cargo.toml

```
@@ -19,8 +19,8 @@ forest_encoding = "0.2.1"
 sha2 = { version = "0.9", default-features = false }
 byteorder = "1.3.4"
 async-trait = "0.1"
-base64 = { version = "0.13", optional = true }
-forest_json_utils = { version = "0.1", optional = true }
+base64 = { version = "0.13"}
```

we need some `toml` formatter.
```suggestion
base64 = "0.13"
```